src/test/java/uk/co/theautomatedtester/book/HomePageBookTest.java

```diff
@@ -163,4 +164,9 @@ public void testClicksAllLinksOpenInNewWindow() throws InterruptedException {
     public void tearDown() {
         webDriver.quit();
     }
+
+    @AfterTest
+    public void tearDownTest(){
```

Comment:
are you sure you want to comment empty method?

Reply:
Yes